lib/ruby_smb/smb1/packet/trans.rb

```diff
@@ -0,0 +1,16 @@
+module RubySMB
```

Comment:
@chrisdlf has a branch right now that currently also implements trans/trans2 packets in a very similar way to this, also including spec tests with a few minor differences. We like your implementation and wanted to see what your thoughts were: https://github.com/UserExistsError/ruby_smb/compare/simpleclient-progress...chrisdlf:feature/100/dcerpc_smb1_named_pipe?expand=1#diff-42e45c6d8a4c2b5cd3c864fcccc34337 . We can modify his branch to work with this, we just wanted to ask if you had any thoughts about the different approaches.

Reply:
That implementation looks much more complete. I renamed a few things in my last commit to follow naming conventions used in that branch.